self-core-impl/src/test/java/com/selfxdsd/core/mock/InMemoryProjectManagersTestCase.java

```diff
@@ -0,0 +1,66 @@
+package com.selfxdsd.core.mock;
+
+import com.selfxdsd.api.storage.Storage;
+import org.junit.Test;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.is;
+
+/**
+ * Unit tests for class {@link InMemoryProjectManagers}.
+ *
+ * @author hpetrila
+ * @version $Id$
+ * @todo #82:5min Rename InMemoryTestCase to InMemoryUsersTestCase.
+ * @since 0.0.1
+ */
+
+public final class InMemoryProjectManagersTestCase {
+    /**
+     * Get project manager by id.
+     */
+    @Test
+    public void getProjectManagerById() {
+        final Storage storage = new InMemory();
+        InMemoryProjectManagers inMemoryPm =
+                new InMemoryProjectManagers(storage);
+        assertThat(inMemoryPm.getById(1),
+                is(inMemoryPm.pick("github")));
```

Comment:
@HoreaPetrila testul asta nu e corect. Functiei ``is(...)`` trebuie tu sa ii dai PM-ul expected, nu sa chemi alta metoda tin InMemoryProjectManagers. 

Daca testul asta, asa cum e el acuma crapa, nu vei stii daca a problema cu ``getById`` sau cu ``pick(...)``. 

Poti face assert pe baza id-ului din PM.